alphagov/paas-admin
# Replace jsdom in svg graph generation

What
----

`jsdom` [depends on now deprecated packages](https://github.com/jsdom/jsdom/issues/3030) so we looked to [temporarily list dependency directly to have a working build](https://github.com/alphagov/paas-admin/pull/1172)
By removing jsdom from production dependency, we no longer need to do that.

[Happy DOM](https://github.com/capricorn86/happy-dom/) is a lighter, faster dom environment.

How to review
-------------

- deploy to own dev or build locally
- check graphs render fine
- review code
- agree/disagree with this change

Who can review
---------------

not @kr8n3r
